File: app/src/main/java/com/cliff/conch/box/scene/ActivityTaskManager.java
package com.cliff.conch.box.scene;

import com.cliff.conch.box.reflect.RefClass;
import com.cliff.conch.box.reflect.RefStaticMethod;
import com.cliff.conch.box.reflect.RefStaticObject;

public class ActivityTaskManager {
    public static Class<?> TYPE = RefClass.load(ActivityTaskManager.class, "android.app.ActivityTaskManager");
    public static RefStaticObject<Singleton> IActivityTaskManagerSingleton;
    public static RefStaticMethod<?> getService;
}

File: reflect-android/src/main/java/reflect/android/app/ActivityTaskManager.kt
package reflect.android.app

import com.cliff.reflection.common.annotation.PStaticField
import com.cliff.reflection.common.annotation.PStaticMethod
import com.cliff.reflection.common.annotation.ProxyClass

@ProxyClass("android.app.ActivityTaskManager")
interface ActivityTaskManager {
    @PStaticMethod("android.app.IActivityTaskManager")
    fun getService():Any?

    @PStaticField("android.util.Singleton")
    val IActivityTaskManagerSingleton: Any
}